tall tree-sitter-cli

```diff
@@ -0,0 +1,981 @@
+[33mcommit 61100e64ac38ae087ebaa49633f2cb45e49ce3f8[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```

Comment:
most probably this file shouldn't be here

Reply:
good catch, my touchpad I giving me some problem and something it made random stuff.